bot/seasons/valentines/valentine_zodiac.py

```diff
@@ -0,0 +1,55 @@
+import logging
+import random
+from json import load
+from pathlib import Path
+
+import discord
+from discord.ext import commands
+
+from bot.constants import Colours
+
+log = logging.getLogger(__name__)
+
+LETTER_EMOJI = ':love_letter:'
+HEART_EMOJIS = [":heart:", ":gift_heart:", ":revolving_hearts:", ":sparkling_heart:", ":two_hearts:"]
+
+
+class ValentineZodiac:
+    """
+    A cog that returns a counter compatible zodiac sign to the given user's zodiac sign.
+    """
+    def __init__(self, bot):
+        self.bot = bot
+
+    @commands.command(name="partnerzodiac")
+    async def counter_zodiac(self, ctx, zodiac_sign):
+        """
+        Provides a counter compatible zodiac sign to the given user's zodiac sign.
+        """
+
+        with open(Path('bot', 'resources', 'valentines', 'zodiac_compatibility.json'), 'r', encoding="utf8") as f:
```

Comment:
Since this is coming from a static file there's no reason for it to be loaded every time the command is called. Move it to the cog's `__init__` or into the main function's namespace.

Reply:
done